backend/models/chat.py

```diff
@@ -25,6 +25,9 @@ class Platform_Enum(enum.Enum):
 class Chat_Status_Enum(enum.Enum):
     UNREAD = "UNREAD"
     READ = "READ"
+    # TODO ::
```

Comment:
This is the idea that I propose to solve the 24hr window message, I add TODO:: to mark if we will implement it. Since we have different approach for now, I will remove it later.